Dockerfile

```diff
@@ -178,11 +178,12 @@ COPY rootfs/rootfs $ROOTFS
 
 # Build the Hyper-V KVP Daemon
 RUN cd /linux-kernel && \
-    make headers_install INSTALL_HDR_PATH=/usr && \
+    make INSTALL_HDR_PATH=/tmp/kheaders headers_install && \
     cd /linux-kernel/tools/hv && \
-    sed -i 's/\(^CFLAGS = .*\)/\1 -m32/' Makefile && \
+    sed -i 's/\(^CFLAGS = .*\)/\1 -m32 -I\/tmp\/kheaders\/include/' Makefile && \
```

Comment:
Ping @tianon. Should I change the separators in the PR?
If replacing the separator, any preference for a symbol? `!` as in your example?